doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -627,11 +627,17 @@ This section describes some special purpose tactics to work with
       type of the hypothesis referred to by :n:`@natural` has uninstantiated
       parameters, these parameters are left as existential variables.
 
+   .. tacn:: simple injection {? @induction_arg }
+
+      Similar to :tacn:`injection`, but always adds the derived equalities
+      as new :term:`premises <premise>` in the current goal (instead of as
+      new hypotheses) even if the :flag:`Structural Injection` flag is set.
+
    .. flag:: Structural Injection
 
-      This :term:`flag` ensures that :n:`injection @term` erases the original hypothesis
-      and leaves the generated equalities in the context rather than adding them
-      to the current goal as :term:`premises <premise>`, as if giving :n:`injection @term as`
+      When set, :n:`injection @term` adds the generated equalities as
```

Comment:
It would be better to keep the glossary definition: "When this `flag` is set, ..."

Reply:
i.e. to be clear it doesn't refer to "injection ...",  Sure.